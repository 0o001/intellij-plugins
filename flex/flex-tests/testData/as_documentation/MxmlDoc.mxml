<?xml version="1.0" encoding="utf-8"?>

<mx:Application xmlns:mx="http://www.adobe.com/2006/mxml">
  <mx:DataGrid id="dataGrid" width="100%" height="100%" resize<caret>Effect="aa"/>
</mx:Application>