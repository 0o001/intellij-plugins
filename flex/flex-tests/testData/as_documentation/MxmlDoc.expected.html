<div class='definition'><pre>resizeEffect</pre></div><div class='content'><p> Color of text in the component, including the component label.</div><table class='sections'><tr><td valign='top' class='section'><p>Default:</td><td><p><code style='background-color:#0B333C; color: #ffffff'>0x0B333C</code></table>